src/test/script_tests.cpp

```diff
@@ -1361,6 +1361,12 @@ static CScript ScriptFromHex(const std::string& str)
     return CScript(data.begin(), data.end());
 }
 
+template <size_t N>
+CScript ToScript(const std::array<uint8_t, N>& array)
```

Comment:
Are you suggesting adding `MakeUCharSpan` so that one could send in a raw array that would otherwise not have `.data()` and `.size()`?

Wouldn't the call to `MakeUCharSpan` have to be re-touched if we switch to using only `std::byte` for `CScript`?

Reply:
> No, just to hide the reinterpret cast behind a curtain

Aha, so you are concerned about the element type of the container mismatching, but still within allowable realm of `UCharCast`, got it.

Wait.. that will allow me to change `ToScript(HexLiteral<uint8_t>(` into `ToScript(HexLiteral(` - thanks!